packages/monaco/src/prettier.ts

```diff
@@ -0,0 +1,34 @@
+import prettier from 'prettier/standalone';
+import estree from 'prettier/plugins/estree';
+import typescript from 'prettier/plugins/typescript';
+import type { Options } from 'prettier';
+import type * as monaco from 'monaco-editor';
+
+// TODO: use the settings store for the config
+const prettierConfig = {
+  plugins: [estree, typescript],
+  parser: 'typescript',
+  useTabs: true,
```

Comment:
controversial setting 😬 

Maybe you could make this use the current editor setting?

Reply:
I think we already control the tab size with the `settings-store` so it shouldn't be an issue

Plus this saves us many bytes of space in the db

Reply:
oh we don't even support spaces lmao
<img width="832" alt="image" src="https://github.com/bautistaaa/typehero/assets/996134/407fa69b-6cc8-4a2e-90d0-05b0c3d701f9">

Reply:
I think we support whatever the editor uses which might default to spaces